src/webgl/p5.Camera.js

```diff
@@ -193,6 +193,23 @@ p5.prototype.perspective = function (...args) {
   return this;
 };
 
+/**
+ *
+ * Enable or disable perspective for lines in the WebGL renderer.
+ * When linePerspective is enabled, lines will be affected by the current camera's perspective.
+ * When linePerspective is disabled, lines will have a uniform scale regardless of the camera's perspective.
+ *
+ * @method linePerspective
+ * @memberof p5.prototype
+ * @param {boolean} enable - Set to `true` to enable line perspective, `false` to disable.
+ *<br>
+ * @example
+ * @todo
+ */
+p5.prototype.linePerspective = function (enable) {
```

Comment:
Can we add a second overload to this function that has no parameters and returns the current value? Similar to `frameRate`

Reply:
@davepagurek do you want me to validate its parameters too or this would work fine?
```
p5.prototype.linePerspective = function (enable) {
  if (enable !== undefined) {
    // Set the line perspective if enable is provided
    this._renderer._curCamera.useLinePerspective = enable;
  } else {
    // If no argument is provided, return the current value
    return this._renderer._curCamera.useLinePerspective;
  }
};
```

Reply:
Validating parameters would be great too! Looks good so far.